Fix break with LAYER_BASED_SVG_ENGINE disabled after 271550@main

ykimot opened the pull request:
#### 4b8a40f948e610010bc2f931f431ed69c0c2d48e
<pre>
Fix break with LAYER_BASED_SVG_ENGINE disabled after 271550@main
<a href="https://bugs.webkit.org/show_bug.cgi?id=265986">https://bugs.webkit.org/show_bug.cgi?id=265986</a>

Reviewed by Don Olmstead.

&quot;context&quot; in RenderSVGInline::rectsForRepaintingAfterLayout() is no longer used.
To fix the build break, deleting the line.

* Source/WebCore/rendering/svg/RenderSVGInline.cpp: deleted a line that uses &quot;context&quot;.
(WebCore::RenderSVGInline::clippedOverflowRect const):

Canonical link: <a href="https://commits.webkit.org/271775@main">https://commits.webkit.org/271775@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1910e82ca71dcc872ce87bd602dda95fc5ae3c02

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/29370 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/8037 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/30698 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/31934 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/26664 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/29970 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/10172 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/5331 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/26661 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/29643 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/6679 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/25118 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/5746 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/5883 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/26174 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/33275 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/26802 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/26604 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/32101 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/5847 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/4029 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/29884 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/7550 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/7029 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/6372 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/6371 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

foopoiuyt commented on Source/WebCore/rendering/svg/RenderSVGInline.cpp:
```diff
@@ -79,8 +79,6 @@ LayoutRect RenderSVGInline::clippedOverflowRect(const RenderLayerModelObject* re
 #if ENABLE(LAYER_BASED_SVG_ENGINE)
     if (document().settings().layerBasedSVGEngineEnabled())
         return RenderInline::clippedOverflowRect(repaintContainer, context);
-#else
```
While this seems unnecessary since context is used on the next real line, the error message I see from our internal build seems to imply the error is on line 94 which is the UNUSED_PARAM in the next method not this one.

ykimot replied:
Thanks! Oops, my fix was wrong, will fix it

ykimot replied:
Fixed.

ykimot committed:
Fix break with LAYER_BASED_SVG_ENGINE disabled after 271550@main
https://bugs.webkit.org/show_bug.cgi?id=265986

Reviewed by Don Olmstead.

"context" in RenderSVGInline::rectsForRepaintingAfterLayout() is no longer used.
To fix the build break, deleting the line.

* Source/WebCore/rendering/svg/RenderSVGInline.cpp: deleted a line that uses "context".
(WebCore::RenderSVGInline::clippedOverflowRect const):

Canonical link: https://commits.webkit.org/271775@main
